### PR TITLE
Update notrana.json

### PR DESCRIPTION
**Pull request by Zer0Rea1**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/notrana.json [hunk 1]
```diff
@@ -4,9 +4,11 @@
...
     },
     "record": {
-        "A": ["217.174.245.249"],
-        "AAAA": ["2a00:da00:1800:83a4::1"],
-        "MX": ["hosts.is-a.dev"],
-        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all"
+        "CNAME": "notrana.github.io",
+        "MX": [
+            "mx1.improvmx.com.",
+            "mx2.improvmx.com."
```
Comment (WilliamDavidHarrison) [inline domains/notrana.json:10]:
```suggestion
            "mx1.improvmx.com",
            "mx2.improvmx.com"
```

File: domains/notrana.json [hunk 2]
```diff
@@ -4,9 +4,11 @@
         "email": "benyameen75@gmail.com"
     },
     "record": {
-        "A": ["217.174.245.249"],
-        "AAAA": ["2a00:da00:1800:83a4::1"],
-        "MX": ["hosts.is-a.dev"],
-        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all"
+        "CNAME": "notrana.github.io",
```
Comment (WilliamDavidHarrison) [inline domains/notrana.json:7]: You need to use A records here.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/notrana.json (hunk 2 above)
Comment (NotRana) [reply domains/notrana.json:7]: ok

**Review by Zer0Rea1 — COMMENTED** [review]
Changed

**Pull request closed without merging**